rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/RMDBManager.java

```diff
@@ -333,36 +335,39 @@ public NodeSourceData doInTransaction(Session session) {
     }
 
     public void updateNodeSource(final NodeSourceData nodeSourceData) {
+        logger.debug("Request " + RMDBManagerBuffer.class.getSimpleName() + " to update node source " +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Request " 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)